roles/common/handlers/main.yml

```diff
@@ -16,3 +13,16 @@
   failed_when: logrotate_test.rc != 0
   when:
     - running_on_server
+
+- name: reload logrotate timer settings
+  ansible.builtin.systemd:
+    daemon_reload: true
+    name: logrotate.timer
+    enabled: true
+
+- name: restart journald
+  ansible.builtin.systemd:
+    name: systemd-journald
+    state: restarted
+  when:
+    - running_on_server
```

Comment:
We shouldn't need this in a handler - it only runs if some other task returns `changed`, so by definition if you're running a handler you're already failing idempotency.